src/Commands/MeltorCommand.php

```
@@ -0,0 +1,309 @@
+
+        try {
+            $this->schemaConnection = DB::connection($schemaConnectionName);
+        } catch (Exception $exception) {
+            $this->warn(sprintf('Please configure a database connection named %s that points to the information_schema database. See README.md.', $schemaConnectionName));
+            return 1;
+        }
+
+        if ($this->option('restore')) {
+            $this->meltor->restoreBackup($this);
+
+            return 0;
+        }
+
+        $this->newLine();
+        $this->line(sprintf('Meltor tries to consolidate all past migrations, based on this system\'s "%s" DB', $this->databaseName));
+        $this->line('Make sure all your systems have been deployed, and that this system has your current, production database structure.');
+
+        if ($this->option('testrun')) {
+            $this->newLine();
+            $this->line('Test run enabled');
+            if (!$this->option('force')) {
+                $ready = $this->ask(
+                    'Please confirm that you have deleted all old migrations, except ones that change framework or package created tables (y|N)'
+                );
+
+                if ($ready != 'y') {
+                    $this->info('Aborting');
+
+                    return 0;
+                }
+            }
+        }
+
+        $this->newLine();
+        $this->line(sprintf('Reading structure information from connection %s, database %s', $this->meltor->config('connection.data'), $this->databaseName));
+
+        $this->structure   = $this->meltor->getDatabaseStructure($this->databaseName, $this->schemaConnection);
+        $uniqueKeys        = $this->meltor->getUniqueKeys($this->databaseName, $this->schemaConnection);
+        $foreignKeys       = $this->meltor->getForeignKeys($this->databaseName, $this->schemaConnection);
+        $innoDbIndexResult = $this->meltor->getIndexesFromInnoDb($this->databaseName, $this->schemaConnection, 0);
+
+        if (is_array($innoDbIndexResult)) {
```

Returning the exception as a string but not doing anything with it is weird.
Can't you just let the exception throw and do a try...catch here?